boards/arm/mec15xxevb_assy6853/doc/index.rst

```diff
@@ -240,33 +261,21 @@ Flashing
    - Stop bits: 1
 
 #. Connect the MEC15xxEVB_ASSY_6853 board to your host computer using the
-   UART2 port. Then build :ref: `hello_world` application. It is important
-   to generate a binary with a new load address, for example do the following::
-
-        ${OBJCOPY} --change-addresses -0xe0000 -O binary -S ${in_elf} ${out_bin}
-
-   Once you obtain the binary, proceed to use the microchip tool mec15xx_spi_gen
-   in order to create the final binary. This binary is what you need to flash
-   in your spi nor.
-
-   .. zephyr-app-commands::
-      :zephyr-app: samples/hello_world
-      :board: mec15xxevb_assy6853
-      :goals: build flash
+   UART2 port and apply power.
 
-   You should see "Hello World! mec15xxevb_assy6853" in your terminal.
+    You should see ``"Hello World! mec15xxevb_assy6853"`` in your terminal.
```

Comment:
The ``Y`` in ``You`` should be indented the same as the ``U`` in ``UART2`` on the line above.  (The extra space indent changes how the output looks.)

Reply:
done